src/Crypto/JWT.hs

```diff
@@ -227,6 +229,38 @@ instance ToJSON ClaimsSet where
     ] ++ M.toList (filterUnregistered o)
 
 
+-- | Validate the claims made by a ClaimsSet. Currently only inspects
+-- the exp and nbf claims.
+--
+validateClaimsSet
+  :: MonadTime m
+  => State ValidationSettings z
```

Comment:
I see that the `State ValidationSettings z` argument is not being used, but we could leave it for now
because eventually we might want to add "allowable skew" to `ValidationSettings`.